--- origsrc/glew-1.10.0/Makefile	2014-03-23 02:13:56.345606400 -0500
+++ src/glew-1.10.0/Makefile	2014-03-23 04:16:17.045470400 -0500
@@ -134,7 +134,7 @@ glew.pc: glew.pc.in
 		-e "s|@includedir@|$(INCDIR)|g" \
 		-e "s|@version@|$(GLEW_VERSION)|g" \
 		-e "s|@cflags@||g" \
-		-e "s|@libname@|GLEW|g" \
+		-e "s|@libname@|glew32|g" \
 		< $< > $@
 
 # GLEW MX static and shared libraries
@@ -174,7 +174,7 @@ glewmx.pc: glew.pc.in
 		-e "s|@includedir@|$(INCDIR)|g" \
 		-e "s|@version@|$(GLEW_VERSION)|g" \
 		-e "s|@cflags@|-DGLEW_MX|g" \
-		-e "s|@libname@|GLEWmx|g" \
+		-e "s|@libname@|glew32mx|g" \
 		< $< > $@
 
 # GLEW utility programs
--- origsrc/glew-1.10.0/glew.pc.in	2013-07-22 09:11:22.000000000 -0500
+++ src/glew-1.10.0/glew.pc.in	2014-03-23 04:15:45.627673400 -0500
@@ -8,4 +8,4 @@ Description: The OpenGL Extension Wrangl
 Version: @version@
 Cflags: -I${includedir} @cflags@
 Libs: -L${libdir} -l@libname@
-Requires: glu
+Libs.private: -lopengl32
